pkg/apis/serving/v1/revision_types.go

```diff
@@ -88,6 +88,12 @@ type RevisionSpec struct {
 	// (send network traffic). If unspecified, a system default will be provided.
 	// +optional
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty"`
+
+	// IdleTimeoutSeconds is the maximum duration in seconds that the request routing
+	// layer will wait when there is no network traffic from a container. If unspecified,
```

Comment:
Should we be more precise here, like: "is the maximum duration in seconds a request will be allowed to stay open while not receiving any bytes from the user's application"?

Reply:
Good idea! updated